src/utils/constants.ts

```diff
@@ -9,23 +13,122 @@ export const ERC20_TRANSFER_SIGNATURE = '0xa9059cbb';
 export const ERC20_APPROVE_SIGNATURE = '0x095ea7b3';
 export const DEFAULT_TOKEN_DECIMALS = 18;
 
-export const NETWORK_NAMES = {
-  '1': 'mainnet',
-  '4': 'rinkeby',
-  '100': 'xdai',
-  '42161': 'arbitrum',
-  '421611': 'arbitrumTestnet',
-  '1337': 'localhost',
-};
-
 const defaultInfuraKey = process.env.REACT_APP_KEY_INFURA_API_KEY;
-export const DEFAULT_RPC_URLS = {
-  '1': `https://mainnet.infura.io/v3/${defaultInfuraKey}`,
-  '4': `https://rinkeby.infura.io/v3/${defaultInfuraKey}`,
-  '100': 'https://rpc.xdaichain.com/',
-  '42161': `https://arbitrum-mainnet.infura.io/v3/${defaultInfuraKey}`,
-  '421611': `https://arbitrum-rinkeby.infura.io/v3/${defaultInfuraKey}`,
-};
+
+export const NETWORKS: {
+  id: number,
+  name: string,

```

Comment:
wouldn't this be better defined as a type or an interface, and then 
export const NETWORKS: networkType [] ?